src/components/Venues/Modal/VenueModal.tsx

```
@@ -61,282 +27,205 @@
+                  alt="Venue Image Preview"
+                  style={{ maxWidth: '100px', maxHeight: '100px' }}
+                  // temporary bypass of CORS error
+                  crossOrigin="anonymous"
+                />
+                <button
+                  type="button"
+                  onClick={clearImageInput}
+                  data-testid="closeimage"
+                >
+                  <i className="fa fa-times"></i>
+                </button>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)